cmd/engflow_auth/main.go

```diff
@@ -209,6 +209,17 @@ Visit %s for help.`,
 	return nil
 }
 
+func (r *appState) logout(cliCtx *cli.Context) error {
+	if cliCtx.NArg() != 1 {
+		return autherr.CodedErrorf(autherr.CodeBadParams, "expected exactly 1 positional argument, a cluster name")
+	}
+	cluster := cliCtx.Args().Get(0)
```

Comment:
The cluster name isn't sanitized here the same way it is on `login` or other methods, which means that the same string passed to login and logout may not result in a successful logout - maybe add a call to `sanitizedURL` here, and then lookup the token based on `.Host`?

Reply:
Good point, done